src/services/cli.js

```diff
@@ -9,13 +9,13 @@
  *
  * @module cli
  */
-
 /* global fetch */
 require( 'isomorphic-fetch' );
 
 const createLoader = require( './loader' );
 const createRunner = require( './runner' );
 
+const { generateOutputForJenkins } = require( '../reporters/metric.js' );
```

Comment:
Should not mention "Jenkins" directly, instead we can name the output format (JSON).